Gemfile

```diff
@@ -17,6 +17,9 @@ ruby '2.5.1'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'bootsnap', '>= 1.1.0', require: false
```

Comment:
We need to pull this out again at some point.  Ok with that being done later for all the apps that were upgraded.

Reply:
why?